### ISSUE TITLE
How to cite

### ISSUE THREAD

**Issue by MatTakahiro** [m-wrzr/populartimes#39]
Hi, m-wrzr!
Many thanks for your great package.
Could you provide how to cite information of your populartimes package?
I will cite your package in our paper.

Best regards,
Mat

**Comment by rgudis1** [comment]
Hi Mat,
           You mentioned that you were able to get the package to work which implies that you got the data that you needed. Could you please tell me how did you implement the package on your system ? Which version of python did you use ? which operating system did you use ? How did you implement populartimes.get() and populartimes.get_id() ? would also please share your python program where you implemented populartimes.get() and populartimes.get_id() ? Thank you !

**Comment by m-wrzr** [comment]
Hey, I would suggest you cite Google Maps directly. It's their data and corresponds to the displayed maps. If you want to cite the tool too, you could do something like this:

```
@misc{populartimes,
  author = {m-wrzr, riedmaph},
  title = {populartimes},
  year = {2018},
  publisher = {GitHub},
  journal = {GitHub repository},
  howpublished = {\url{https://github.com/m-wrzr/populartimes}},
  commit = {a5205d7fc909ca09dd18ad5de2dcbd0096cfc817}
}
```

Can you post a link to the paper here once it is finished? I would be very interested in the research!

**Comment by MatTakahiro** [comment]
Hi, m-wrzr and rgudis1!
Sorry for my late response.
Thanks all of your great contributions.
Yes! I will paste a link to my (our) paper which will be published.

I simply collected populartime data in a site by using your populartimes.get() function.
Then, I used their coordinate and dynamically mapped them as you show.
Finally, I viewed the data as a quasi-real-time building (shop) activity data.

I am grad to communicate with you, thank you very much!
Mat
